ansible_collections/arista/avd/examples/single-dc-multipod-l3ls/README.md

```diff
@@ -0,0 +1,287 @@
+---
+# This title is used for search results
+title: AVD example for a single data center using multiple pods for l3ls
```

Comment:
Should we use capital POD everywhere?

Reply:
I would recommend leaving it lowercase unless its in a title

Reply:
It is an abbreviation. Same for L3LS.

Reply:
Hmm I always thought of it as a collection of nodes vs an abbreviation. I learned something today.